Tasks/JenkinsDownloadArtifacts/task.json

```diff
@@ -122,6 +127,67 @@
             "visibleRule": "downloadCommitsAndWorkItems == true",
             "groupName": "advanced",
             "helpMarkDown": "Optional file name suffix for commits and workitem attachment. Attachment will be created with commits_{suffix}.json and workitem_{suffix}.json. If this input is not provided attachments will be create with the name commits.json and workitems.json"
+        },
+        {
+            "name": "propagatedArtifacts",
+            "type": "boolean",
+            "label": "Artifacts are propagated to Azure",
+            "defaultValue": "false",
+            "required": false,
+            "groupName": "propagatedArtifactsGroup",
+            "helpMarkDown": "Check this if Jenkins artifacts are propagated."
```

Comment:
A little more help will be good here "Check this if Jenkins artifacts were propagated to Azure. To upload Jenkins artifacts to azure, refer to this Jenkins plugin: <provide link to jenkins azure>"